core/src/ast/types.rs

```diff
@@ -249,6 +306,7 @@ impl TypeName {
     pub fn resolve_with_path<'a>(&self, in_path: &Path, env: &'a Env) -> (Path, &'a CustomType) {
         match self {
             TypeName::Named(local_path) => {
+                let local_path = &local_path.path;
```

Comment:
`local_path` is a `PathType` right? Can we rename it to `local_path_type` for clarity? Otherwise this name shadowing is confusing